### PR TITLE
Mike

### PR DESCRIPTION
**Pull request by mmccree1**
US 14, 19, tests and updated test_suite

### CODE REVIEW

File: test/us19_test.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import unittest
+import sys
+import os
+sys.path.append(os.path.abspath('../src'))
+from us19 import us19
+from Gedcom import Gedcom, Family
```
Comment (ming1in) [inline test/us19_test.py:6]: this import isnt being used

**Review by ming1in — APPROVED** [review]
users stories and tests are good. Need to fix those merge conflict, lmk if ya need help

**Review by samanthainneo99 — APPROVED** [review]
Everything looks fine to me

File: src/us14.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+Created on 10/24/20
+@author:   Michael McCreesh
+Pledge:    I pledge my honor to abide by the Stevens Honor System
+SSW555 - Sprint 3
+'''
+
+'''Goal: Make sure there are no more than 5 births at a time'''
+def us14(gedcomFile):
+    gedcom = Gedcom(gedcomFile)
+    badFams = []
```
Comment (arana23) [inline src/us14.py:11]: lol badFams

**Review by arana23 — APPROVED** [review]
Nice!

**Pull request merged**